src/Microsoft.TemplateEngine.Abstractions/Installer/CheckUpdateResult.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.TemplateEngine.Abstractions.TemplatePackages;
+
+namespace Microsoft.TemplateEngine.Abstractions.Installer
+{
+    public class CheckUpdateResult : Result
+    {
+        public string LatestVersion { get; private set; }
+
+        public bool IsLatestVersion { get; private set; }
```

Comment:
I have documented it now, will be in next commit:
```
        /// <summary>
        /// The latest version for the template package identified as the result of update check.
        /// </summary>
        public string LatestVersion { get; private set; }

        /// <summary>
        /// True when the template package is already at the latest version.
        /// </summary>
        /// <remarks>In some cases the version installed can be higher then identified during update check. Since only installer can correctly compare the versions, the installer returns the result if the version is latest rather than letting the caller determine it using string comparison.</remarks>
        public bool IsLatestVersion { get; private set; }
```